layouts/partials/layout/header.html

```diff
@@ -0,0 +1,19 @@
+{{ $alert := index .Site.Data .Site.Language.Lang }} {{ partial
```

Comment:
Why need for seperate header? Better to reuse header from theme and configure yaml in content as intended?

Reply:
Both language and nav links are not in the new design. Also logo icon and it's format (now both icon and text) are changed. Maybe it would be best to change digdir-hugo-theme before removing this custom header? They are very different